CONTRIBUTING.md

```diff
@@ -102,6 +102,16 @@ This method has the benefit of being able to compile to test your change without
    - This will modify the appropriate patches based on your commits
 6. PR your modifications to github
 
+### Method 3 - Fixup Commits
+This is the automated way of doing Method 2. Basically, you make your changes at HEAD, but use fixup commits to modify earlier patches and git will rebase the commits to the right place and squash them.
+
+1. Make your change while at HEAD
+2. Make a fixup commit. `git commit -a --fixup <hashOfPatchToFix>`. You can get the hash by looking at `git log` or `git blame`, your IDE can assist you too.  
+Alternatively, if you only know the name of the patch, you can do `git commit -a --fixup "Subject of Patch name"`
+3. Rebase with autosquash: `git rebase --autosquash -i upstream/upstream`. It will move your fixup commit to the right place.
+5. Type `./paper rebuild` in the main directory
```

Comment:
This is supposed to be 4. (You can also just use `1.` on all of them and the renderer will handle it for you)